src/acl_mem.cpp

```diff
@@ -965,6 +965,16 @@ CL_API_ENTRY cl_mem CL_API_CALL clCreateBufferIntelFPGA(cl_context context,
                                            errcode_ret);
 }
 
+// Create a buffer with property
```

Comment:
Nitpick: how about the upstream description, `Create a buffer with additional properties`?

Reply:
Thanks I have changed the comment and removed the other unnecessary comment.